### PR TITLE
Display more elaborate message in the error for comparison of subjects present in participants.tsv and the ones present in the BIDS layout

### PR DESCRIPTION
**Pull request by cmadjar**
This PR improves a little the logs of the error when doing comparison between the list of subjects present in participants.tsv and the list of subjects grepped by the BIDS reader.

### CODE REVIEW

**Comment by cmadjar** [comment]
@laemtl If you have some time to review this one. Should be a relatively quick one.

File: python/lib/bidsreader.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def candidates_list_validation(self, participants_info):
 
         mismatch_message = "\nERROR: Participant ID mismatch between " \
                            "participants.tsv and raw data found in the BIDS " \
```
Comment (vietdhoang) [inline python/lib/bidsreader.py:137]:
```suggestion
        mismatch_message = ("\nERROR: Participant ID mismatch between "
                            "participants.tsv and raw data found in the BIDS "
```

File: python/lib/bidsreader.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ def candidates_list_validation(self, participants_info):
...
-                           "directory/n"
+                           "directory\n"
```
Comment (vietdhoang) [inline python/lib/bidsreader.py:138]:
```suggestion
                            "directory")
```

**Review by vietdhoang — CHANGES_REQUESTED** [review]
I added some suggestions to account for the fact that `print()` automatically adds a newline at the end.

I also added a suggestion for how a multi-line string can be formatted. This suggestion can be ignored if the current format is preferred or is the standard at LORIS

**Comment by cmadjar** [comment]
@vietdhoang thank you! I committed the changes. :)

**Pull request merged**